ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,78 +16,288 @@
 
 package io.confluent.ksql.rest.server;
 
+import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.tree.CreateStream;
+import io.confluent.ksql.parser.tree.CreateStreamAsSelect;
+import io.confluent.ksql.parser.tree.CreateTable;
+import io.confluent.ksql.parser.tree.CreateTableAsSelect;
+import io.confluent.ksql.parser.tree.InsertInto;
+import io.confluent.ksql.parser.tree.QualifiedName;
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SetProperty;
+import io.confluent.ksql.parser.tree.Statement;
+import io.confluent.ksql.parser.tree.UnsetProperty;
+import io.confluent.ksql.serde.DataSource.DataSourceType;
 import io.confluent.ksql.util.KsqlConfig;
+import io.confluent.ksql.util.KsqlException;
+import io.confluent.ksql.util.Pair;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 import org.apache.kafka.test.TestUtils;
 import org.easymock.EasyMock;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
 import java.util.Collections;
 
 import io.confluent.ksql.KsqlEngine;
 import io.confluent.ksql.function.UdfLoader;
 import io.confluent.ksql.util.PersistentQueryMetadata;
 import io.confluent.ksql.util.QueryMetadata;
 
+import static org.easymock.EasyMock.anyObject;
 import static org.easymock.EasyMock.anyString;
 import static org.easymock.EasyMock.eq;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
 public class StandaloneExecutorTest {
 
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final KsqlEngine engine = EasyMock.niceMock(KsqlEngine.class);
   private final UdfLoader udfLoader = EasyMock.niceMock(UdfLoader.class);
-  private final String query = "select * from bar;";
-  private StandaloneExecutor executor;
+  private String queriesFile;
 
   @Before
   public void before() throws IOException {
-    final String queriesFile = TestUtils.tempFile().getPath();
-    executor = new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
-    try(final FileOutputStream out = new FileOutputStream(queriesFile)) {
-      out.write(query.getBytes(StandardCharsets.UTF_8));
-    }
+    queriesFile = TestUtils.tempFile().getPath();
+  }
+
+
+  @Test
+  public void shouldRunCsStatement() throws IOException {
+
+    final StandaloneExecutor standaloneExecutor =
```

Comment:
There is a lot of repeated code in these tests - consider moving a lot of the initialisation into the `before()` method e.g. `standaloneExecutor`, `metaStore`, `props`, etc.

Reply:
Moved them to before().